api/cmd/bootstrap.go

```diff
@@ -74,5 +74,6 @@ func startKetoBootstrap(bootstrapCfg *BootstrapConfig) error {
 	updateRequest := enforcer.NewAuthorizationUpdateRequest()
 	updateRequest.SetRoleMembers(enforcer.MLPProjectsReaderRole, bootstrapCfg.ProjectReaders)
 	updateRequest.SetRoleMembers(enforcer.MLPAdminRole, bootstrapCfg.MLPAdmins)
+	updateRequest.AddRolePermissions(enforcer.MLPAdminRole, []string{"mlp.projects.post"})
```

Comment:
As there could be multiple permissions that need to be assigned in the future, it's probably better if we assigned the list of permissions to a variable, with good naming to demonstrate the intention that these are the default permissions granted to an MLP administrator.